src/Indicator.vala

```diff
@@ -27,14 +27,45 @@ public class Power.Indicator : Wingpanel.Indicator {
     private Services.Device primary_battery;
     private bool notify_battery = false;
 
+    private Widgets.ScreenBrightness screen_brightness;
+
     public Indicator (bool is_in_session) {
         Object (code_name : Wingpanel.Indicator.POWER,
                 display_name : _("Power"),
                 description: _("Power indicator"));
 
         this.is_in_session = is_in_session;
```

Comment:
Can you remove the `this.` from here and from the entire file while we're at it

Reply:
Ok, but I didn't touched `this.is_in_session = is_in_session;`, because I don't know what it really does.